validator/src/main.rs

```diff
@@ -639,6 +639,12 @@ pub fn main() {
                 .takes_value(false)
                 .help("Use the RPC service of trusted validators only")
         )
+        .arg(
+            Arg::with_name("no_db_compaction")
+                .long("no-db-compaction")
```

Comment:
I think I'd rather this be `--no-rocksdb-compaction` to be more precise, and to make it obvious that we can remove it if/when rocksdb 🗑